Traceback (most recent call last):
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/adam/projects/website/env/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/adam/projects/website/env/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------


import nltk

# nltk.download("stopwords")
# nltk.download("punkt")
from nltk.corpus import stopwords

stop_words = set(stopwords.words("english"))
words = nltk.word_tokenize(text)
filtered_lowercase_words = []
for word in words:
    word = word.lower()
    if word not in stop_words:
        filtered_lowercase_words.append(word)
filtered_lowercase_words[:12]
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mLookupError[0m                               Traceback (most recent call last)
File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/corpus/util.py:84[0m, in [0;36mLazyCorpusLoader.__load[0;34m(self)[0m
[1;32m     83[0m [38;5;28;01mtry[39;00m:
[0;32m---> 84[0m     root [38;5;241m=[39m [43mnltk[49m[38;5;241;43m.[39;49m[43mdata[49m[38;5;241;43m.[39;49m[43mfind[49m[43m([49m[38;5;124;43mf[39;49m[38;5;124;43m"[39;49m[38;5;132;43;01m{[39;49;00m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43msubdir[49m[38;5;132;43;01m}[39;49;00m[38;5;124;43m/[39;49m[38;5;132;43;01m{[39;49;00m[43mzip_name[49m[38;5;132;43;01m}[39;49;00m[38;5;124;43m"[39;49m[43m)[49m
[1;32m     85[0m [38;5;28;01mexcept[39;00m [38;5;167;01mLookupError[39;00m:

File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/data.py:583[0m, in [0;36mfind[0;34m(resource_name, paths)[0m
[1;32m    582[0m resource_not_found [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00msep[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00mmsg[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00msep[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;124m"[39m
[0;32m--> 583[0m [38;5;28;01mraise[39;00m [38;5;167;01mLookupError[39;00m(resource_not_found)

[0;31mLookupError[0m: 
**********************************************************************
  Resource [93mstopwords[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('stopwords')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mcorpora/stopwords.zip/stopwords/[0m

  Searched in:
    - '/home/adam/nltk_data'
    - '/home/adam/projects/website/env/nltk_data'
    - '/home/adam/projects/website/env/share/nltk_data'
    - '/home/adam/projects/website/env/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************


During handling of the above exception, another exception occurred:

[0;31mLookupError[0m                               Traceback (most recent call last)
Cell [0;32mIn [13], line 7[0m
[1;32m      3[0m [38;5;66;03m# nltk.download("stopwords")[39;00m
[1;32m      4[0m [38;5;66;03m# nltk.download("punkt")[39;00m
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnltk[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcorpus[39;00m [38;5;28;01mimport[39;00m stopwords
[0;32m----> 7[0m stop_words [38;5;241m=[39m [38;5;28mset[39m([43mstopwords[49m[38;5;241;43m.[39;49m[43mwords[49m([38;5;124m"[39m[38;5;124menglish[39m[38;5;124m"[39m))
[1;32m      8[0m words [38;5;241m=[39m nltk[38;5;241m.[39mword_tokenize(text)
[1;32m      9[0m filtered_lowercase_words [38;5;241m=[39m []

File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/corpus/util.py:121[0m, in [0;36mLazyCorpusLoader.__getattr__[0;34m(self, attr)[0m
[1;32m    118[0m [38;5;28;01mif[39;00m attr [38;5;241m==[39m [38;5;124m"[39m[38;5;124m__bases__[39m[38;5;124m"[39m:
[1;32m    119[0m     [38;5;28;01mraise[39;00m [38;5;167;01mAttributeError[39;00m([38;5;124m"[39m[38;5;124mLazyCorpusLoader object has no attribute [39m[38;5;124m'[39m[38;5;124m__bases__[39m[38;5;124m'[39m[38;5;124m"[39m)
[0;32m--> 121[0m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m__load[49m[43m([49m[43m)[49m
[1;32m    122[0m [38;5;66;03m# This looks circular, but its not, since __load() changes our[39;00m
[1;32m    123[0m [38;5;66;03m# __class__ to something new:[39;00m
[1;32m    124[0m [38;5;28;01mreturn[39;00m [38;5;28mgetattr[39m([38;5;28mself[39m, attr)

File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/corpus/util.py:86[0m, in [0;36mLazyCorpusLoader.__load[0;34m(self)[0m
[1;32m     84[0m             root [38;5;241m=[39m nltk[38;5;241m.[39mdata[38;5;241m.[39mfind([38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00m[38;5;28mself[39m[38;5;241m.[39msubdir[38;5;132;01m}[39;00m[38;5;124m/[39m[38;5;132;01m{[39;00mzip_name[38;5;132;01m}[39;00m[38;5;124m"[39m)
[1;32m     85[0m         [38;5;28;01mexcept[39;00m [38;5;167;01mLookupError[39;00m:
[0;32m---> 86[0m             [38;5;28;01mraise[39;00m e
[1;32m     88[0m [38;5;66;03m# Load the corpus.[39;00m
[1;32m     89[0m corpus [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39m__reader_cls(root, [38;5;241m*[39m[38;5;28mself[39m[38;5;241m.[39m__args, [38;5;241m*[39m[38;5;241m*[39m[38;5;28mself[39m[38;5;241m.[39m__kwargs)

File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/corpus/util.py:81[0m, in [0;36mLazyCorpusLoader.__load[0;34m(self)[0m
[1;32m     79[0m [38;5;28;01melse[39;00m:
[1;32m     80[0m     [38;5;28;01mtry[39;00m:
[0;32m---> 81[0m         root [38;5;241m=[39m [43mnltk[49m[38;5;241;43m.[39;49m[43mdata[49m[38;5;241;43m.[39;49m[43mfind[49m[43m([49m[38;5;124;43mf[39;49m[38;5;124;43m"[39;49m[38;5;132;43;01m{[39;49;00m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43msubdir[49m[38;5;132;43;01m}[39;49;00m[38;5;124;43m/[39;49m[38;5;132;43;01m{[39;49;00m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m__name[49m[38;5;132;43;01m}[39;49;00m[38;5;124;43m"[39;49m[43m)[49m
[1;32m     82[0m     [38;5;28;01mexcept[39;00m [38;5;167;01mLookupError[39;00m [38;5;28;01mas[39;00m e:
[1;32m     83[0m         [38;5;28;01mtry[39;00m:

File [0;32m~/projects/website/env/lib/python3.9/site-packages/nltk/data.py:583[0m, in [0;36mfind[0;34m(resource_name, paths)[0m
[1;32m    581[0m sep [38;5;241m=[39m [38;5;124m"[39m[38;5;124m*[39m[38;5;124m"[39m [38;5;241m*[39m [38;5;241m70[39m
[1;32m    582[0m resource_not_found [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00msep[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00mmsg[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m{[39;00msep[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;124m"[39m
[0;32m--> 583[0m [38;5;28;01mraise[39;00m [38;5;167;01mLookupError[39;00m(resource_not_found)

[0;31mLookupError[0m: 
**********************************************************************
  Resource [93mstopwords[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('stopwords')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mcorpora/stopwords[0m

  Searched in:
    - '/home/adam/nltk_data'
    - '/home/adam/projects/website/env/nltk_data'
    - '/home/adam/projects/website/env/share/nltk_data'
    - '/home/adam/projects/website/env/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

LookupError: 
**********************************************************************
  Resource [93mstopwords[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('stopwords')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mcorpora/stopwords[0m

  Searched in:
    - '/home/adam/nltk_data'
    - '/home/adam/projects/website/env/nltk_data'
    - '/home/adam/projects/website/env/share/nltk_data'
    - '/home/adam/projects/website/env/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************


